packages/ui-components/src/components/radio-button-group/radio-button-group.tsx

```diff
@@ -0,0 +1,32 @@
+import { Component, h, Prop, Host, Listen, Event, EventEmitter } from '@stencil/core';
+import { IRadioButton } from '../radio-button/radio-button.types';
+
+@Component({
+	tag: 'kv-radio-button-group',
+	styleUrl: 'radio-button-group.scss',
+	shadow: true
+})
+export class KvRadioButtonGroup {
+	/** (required) List of radio buttons */
+	@Prop({ reflect: true }) buttons: { [key: string]: IRadioButton } = {};
+
+	/** When the radio button selection changes, emit the requested tab's key */
+	@Event() radioButtonChange: EventEmitter<string>;
+
+	/** Listen to custom DOM event of radio button selection */
+	@Listen('checkedChange')
+	radioChangeHandler(event: CustomEvent<string>) {
+		this.buttons[event.detail].active = !this.buttons[event.detail].active;
```

Comment:
If we change the `active` value here, we need to do in `kv-radio-button` the same change?